src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -49,13 +86,245 @@ public Navigator() : base()
         {
             Layout = new AbsoluteLayout();
         }
+        
+        /// <summary>
+        /// An event for the page disappearing signal which can be used to subscribe or unsubscribe the event handler provided by the user.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<TransitionFinishedEventArgs> TransitionFinished;
 
         /// <summary>
         /// List of pages of Navigator.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public List<Page> NavigationPages { get; } = new List<Page>();
 
+        /// <summary>
+        /// Transition properties for the transition of View pair those have same transition tag.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Transition Transition
+        {
+            set
+            {
+                transition = value;
+            }
+            get
+            {
+                return transition;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool ShowBehindPage
+        {
+            set
+            {
+                showBehindPage = value;
+            }
+            get
+            {
+                return showBehindPage;
+            }
+        }
+
+        /// do not traverse child if the view have transitionWithChild true.
+        internal void RetrieveTaggedViews(List<View> taggedViewsInNewPage, View view)
```

Comment:
Please place internal methods after public methods.

Reply:
Thank you.